packages/runtime/test/anonymous/tokens.test.ts

```diff
@@ -33,4 +34,12 @@ describe("Anonymous tokens", () => {
         const token = result.value;
         expect(token.content).toStrictEqual(uploadedToken.content);
     });
+
+    test("should catch a personalized token", async () => {
+        const uploadedPersonalizedToken = await uploadOwnToken(transportServices, runtimeService.address);
+        const resultNotIntended = await noLoginRuntime.anonymousServices.tokens.loadPeerToken({
```

Comment:
`resultNotIntended` is a really strange name, what do you want to say by that?

Reply:
there was the id-and-key test before and now that it's gone, this name doesn't have any purpose anymore

Reply:
please rename it.